packages/fluentui/docs/package.json

```diff
@@ -20,6 +20,7 @@
     "@fluentui/react-northstar-prototypes": "^0.57.0",
     "@fluentui/react-northstar-styles-renderer": "^0.57.0",
     "@fluentui/react-telemetry": "^0.57.0",
+    "@fluentui/scripts": "^1.0.0",
```

Comment:
Oops, please revert this change. We don't want to have `scripts` in production deps 😅

Reply:
Reverted.

But I need help here. So you know why I'm getting a lint error "'@fluentui/scripts' should be listed in the project's dependencies, not devDependencies." when I try to import it to screener steps file?

Reply:
Nevermind :)